specification/automation/resource-manager/Microsoft.Automation/preview/2023-05-15-preview/certificate.json

```diff
@@ -0,0 +1,452 @@
+            "schema": {
+              "$ref": "../../common/v1/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Certificate"
+        ],
+        "operationId": "Certificate_Get",
+        "description": "Retrieve the certificate identified by certificate name.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/certificateoperations"
+        },
+        "x-ms-examples": {
+          "Get a certificate": {
+            "$ref": "./examples/getCertificate.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/AutomationAccountNameParameter"
+          },
+          {
+            "name": "certificateName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of certificate."
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Certificate"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../common/v1/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Certificate"
+        ],
+        "operationId": "Certificate_CreateOrUpdate",
+        "description": "Create a certificate.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/certificateoperations"
+        },
+        "x-ms-examples": {
+          "Create or update a certificate": {
+            "$ref": "./examples/createOrUpdateCertificate.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/AutomationAccountNameParameter"
+          },
+          {
+            "name": "certificateName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The parameters supplied to the create or update certificate operation."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CertificateCreateOrUpdateParameters"
```

Comment:
>CertificateCreateOrUpdateParameters [](http://example.com/codeflow?start=37&length=35)

Any reason why PUT object and GET object are not the same?

Reply:
We have not modified this resource. It came as part of the last stable version.